src/deepsparse/utils/onnx.py

```diff
@@ -472,3 +473,22 @@ def truncate_onnx_embedding_model(
     )
 
     return output_filepath, tmp_file
+
+
+def default_cached_outputs(model_path):
+    """
+    :param model_path: Path to a model
+    :return A list of bools that indicates caching of all outputs except the first one.
+    """
+
+    outputs = list(onnx.load(model_path).graph.output)
```

Comment:
if a model has overridden input shapes is that going to affect the outputs here?

Reply:
nope, it's good we can just look at the outputs